clarifai/runners/dockerfile_template/Dockerfile.nim

```diff
@@ -0,0 +1,71 @@
+FROM nvcr.io/nim/meta/llama-3.1-8b-instruct:1.1.2 as build
+
+FROM gcr.io/distroless/python3-debian12:debug
+
+
+COPY --from=build /bin/bash /bin/bash
+COPY --from=build /bin/sh /bin/sh
+
+# we have to overwrite the python3 binary that the distroless image uses
+COPY --from=build /opt/nim/llm/.venv/bin/python3.10 /usr/bin/python3
+COPY --from=build /opt/nim/llm/.venv/bin/python3.10 /usr/local/bin/python3.10
```

Comment:
we dont need these if we're copying in the entire `/opt` directory L:26

Reply:
oh right, will remove this command